### ISSUE TITLE
tns debug android --log trace not starting app

### ISSUE THREAD

**Issue by CanMehmetK** [NativeScript/nativescript-cli#5591]
tns --version
8.1.3
√ Up to date.

tns debug android --log trace
tns doctor --log trace

locks in same step
Collecting system information...

package.json
'''
{
  "name": "nativescript-sdk-examples-ng",
  "description": "NativeScript & Angular SDK Examples",
  "author": "Telerik <support@telerik.com>",
  "version": "0.1.0",
  "homepage": "https://github.com/NativeScript/nativescript-sdk-examples-ng",
  "repository": {
    "type": "git",
    "url": "https://github.com/NativeScript/nativescript-sdk-examples-ng"
  },
  "bugs": {
    "url": "https://github.com/NativeScript/nativescript-sdk-examples-ng/issues"
  },
  "keywords": [
    "NativeScript",
    "Angular",
    "SDK",
    "example",
    "documentation"
  ],
  "license": "Apache-2.0",
  "dependencies": {
    "@angular/animations": "11.2.1",
    "@angular/common": "11.2.1",
    "@angular/compiler": "11.2.1",
    "@angular/core": "11.2.1",
    "@angular/forms": "11.2.1",
    "@angular/platform-browser": "11.2.1",
    "@angular/platform-browser-dynamic": "11.2.1",
    "@angular/router": "11.2.1",
    "@nativescript/angular": "11.0.1",
    "@nativescript/camera": "5.0.8",
    "@nativescript/core": "7.2.1",
    "@nativescript/geolocation": "7.0.0",
    "@nativescript/theme": "3.0.1",
    "nativescript-intl": "4.0.2",
    "reflect-metadata": "0.1.13",
    "rxjs": "6.6.3",
    "zone.js": "0.11.4"
  },
  "devDependencies": {
    "@angular/compiler-cli": "11.2.1",
    "@nativescript/android": "^7.0.1",
    "@nativescript/ios": "7.2.0",
    "@nativescript/types": "7.2.0",
    "@nativescript/webpack": "4.1.0",
    "@ngtools/webpack": "11.2.1",
    "codelyzer": "6.0.1",
    "fs-extra": "9.1.0",
    "glob": "7.1.6",
    "lazy": "1.0.11",
    "markdown-snippet-injector": "0.2.4",
    "opener": "1.5.2",
    "rimraf": "3.0.2",
    "tar.gz": "1.0.7",
    "tslint": "6.1.3",
    "typescript": "4.1.5"
  },
  "scripts": {
    "pretsc": "npm install",
    "tsc": "tsc",
    "pretslint": "npm run tsc",
    "tslint": "tslint -p tsconfig.json",
    "prebuild": "npm run tslint",
    "build": "node scripts/build.js",
    "postbuild": "npm run inject && npm run archive",
    "inject": "mdinject --root=app --docsroot=dist/code-samples --sourceext=\".ts|.css|.html\" --snippettitles=\"TypeScript|CSS|HTML\"",
    "archive": "node scripts/archive.js",
    "update-webpack": "./node_modules/.bin/update-ns-webpack --deps --configs",
    "update-angular": "./node_modules/.bin/update-app-ng-deps",
    "update-app-ng-deps": "update-app-ng-deps",
    "ns-verify-bundle": "ns-verify-bundle",
    "update-ns-webpack": "update-ns-webpack"
  },
  "main": "main.js"
}

'''

![image](https://user-images.githubusercontent.com/1278985/136673456-d51e566e-f5e5-4b79-9733-aa509baa6c74.png)

**Comment by CanMehmetK** [comment]
@rigor789 Yup, i was in same page but long time wait (more then hour, multiple times, with log trace) nothing happened..

**Comment by dave2k2** [comment]
exactly the same issue here, uninstalled node, uninstalled nativescript... cleared down everything, fresh install of all including java and android sdks, and still hangs. 

bizarrely was working earlier in the day prior to restarting my machine, so something in my environment configuration must be blowing it up. short of completely formatting my machine not sure what else to try... log trace produces almost identically to the OP:

![image](https://user-images.githubusercontent.com/36779990/223165621-80e52259-ec4e-4f45-b884-cf720a4874d7.png)

**Comment by dave2k2** [comment]
Is anyone still struggling with this nightmare try this:

- check your environment variables path for this: C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\run-script\lib\node-gyp-bin 
- remove it 
- restart cmd
- retry tns doctor

for me it completed and resolved this hanging issues, there were no log outputs, was just a pure trial and error guessing game.

**Comment by rigor789** [comment]
@dave2k2 thanks for sharing that!
